wbstack/src/Settings/LocalSettings.php

```diff
@@ -493,7 +489,7 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
 require_once "$IP/extensions/Wikibase/client/ExampleSettings.php";
 
 // Force the concept URIs to be http (as this has always been the way on wbstack)
-$wgWBRepoSettings['conceptBaseUri'] = 'http://' . $wikiInfo->domain . '/entity/';
+$wgWBBaseUri = 'http://' . $wikiInfo->domain . '/entity/';
```

Comment:
[* (repo) (T287486) conceptBaseUri has been removed.](https://github.com/wikimedia/Wikibase/blob/master/RELEASE-NOTES-1.37)

Reply:
The `wg` in the name is for globals, and this isn't a global. I'm pretty sure we can remove the comment above and just move this above the entitySources definition and maybe call the variable something like `$localConceptBaseUri`.